app/league/dashboard/views.py

```diff
@@ -57,6 +57,15 @@ def create_player():
                            player_create_form=form)
 
 
+@blueprint.route('/players/<int:player_id>', methods=['GET'])
+@login_required
+def get_player(player_id):
+    """Get game history and statistics for player"""
+    player = Player.get_by_id(player_id)
+    games = player.games
```

Comment:
Oops, forgot to post this:

Is it necessary to make this separate? Could easily reference `player.games` in the template.